src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,36 @@
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  font-size: 13px;
+  display: inline-flex;
+  color: #e4b400;
+}
+    
+.rating__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+.star {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
All modifiers should work, consider using css code from the previous task

Reply:
still not resolved